.github/renovate.json

```diff
@@ -10,43 +10,19 @@
   ],
   "packageRules": [
     {
-      "matchManagers": ["npm"],
+      "matchManagers": ["npm", "nuget"],
```

Comment:
I think the reason we end up with so many separate .net PRs is because they were not all disabled first?